packages/flutter_tools/lib/src/isolated/native_assets/macos/native_assets_host.dart

```diff
@@ -99,13 +100,13 @@ Future<void> setInstallNamesDylib(
   String newInstallName,
   Map<String, String> oldToNewInstallNames,
 ) async {
-   final ProcessResult setInstallNamesResult = await globals.processManager.run(
+  final ProcessResult setInstallNamesResult = await globals.processManager.run(
     <String>[
       'install_name_tool',
       '-id',
       newInstallName,
-      for (final MapEntry<String, String> entry in oldToNewInstallNames.entries)
-        ...<String>['-change', entry.key, entry.value],
+      for (final MapEntry<String, String> entry in oldToNewInstallNames
```

Comment:
I hate to do this. But it looks like the formatter ran and made a bunch of whitespace changes. (Also in other files.)

I will refer to the Flutter team on if we care about that in the days before turning the formatter on here in the codebase.

Reply:
I've tried to manually revert the auto-formatted lines.